addon/components/notification-message.js

```diff
@@ -1,37 +1,58 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-    classNameBindings: [':c-notification', 'processedType', 'notification.dismiss::c-notification--in', 'autoClear::c-notification--dismissable'],
+  classNames: ['c-notification'],
+  classNameBindings: [
+    'processedType',
+    'notification.dismiss::c-notification--in',
+    'autoClear::c-notification--dismissable'
+  ],
 
-    // Set icon depending on notification type
-    notificationIcon: Ember.computed('notification.type', function() {
-        switch(this.get('notification.type')){
-            case "info":
-                return 'fa-info-circle';
-            case "success":
-                return 'fa-check';
-            case "warning":
-                return 'fa-warning';
-            case "error":
-                return 'fa-exclamation-circle';
-        }
-    }),
+  // Set icon depending on notification type
+  notificationIcon: Ember.computed('notification.type', 'theme', function() {
+    var icons = this.get('icons');
```

Comment:
Hmmm, actually this is correct. However, should it be `Ember.computed('notification.type', 'icons', ...` instead of `theme` ?